src/handlers/common/DefaultRepoRefResolver.ts

```diff
@@ -122,7 +122,7 @@ export class DefaultRepoRefResolver implements RepoRefResolver {
                     branch: opts.branch,
                     rawApiBase: apiUrl,
                 });
-            case ProviderType.bitbucket :
```

Comment:
We need to set editor setting consistently so this kind of meaningless diff doesn't happen